bitnami/etcd/templates/svc.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "etcd.fullname" . }}
+  labels:
+    app: {{ template "etcd.name" . }}
+    chart: {{ template "etcd.chart" . }}
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+  - name: client
+    port: 2379
```

Comment:
Shouldn't we use the values in Values.yaml here?

Reply:
Usually, we have this port hardcoded as it is the port the user will use to access the app and not the port where the app is running (because of the port translation).

We can re-check if this makes sense but we are doing this in all charts